server/middleware/requestLogger.ts

```diff
@@ -0,0 +1,16 @@
+import bunyan, { LoggerOptions } from 'bunyan'
+import type { RequestHandler } from 'express'
+
+export default function createRequestLogger(settings: LoggerOptions): RequestHandler {
```

Comment:
This was the only functionality we were using from the bunyan request logger module.

(That module was no longer maintained)